src/util/stat.cpp

```diff
@@ -1,11 +1,13 @@
-#include <limits>
+#include "util/stat.h"
 
+#include <QDebug>
```

Comment:
why is this needed now?
QtDebug is already included via util/experiment.h in stat.h